demo/demo-jaxrs/jaxrs-client/src/main/resources/microservice.yaml

```diff
@@ -32,3 +32,10 @@ servicecomb:
       name: mycustomrule
     retryEnabled: true
     retryHandler: mycustomhandler
+cse:
+  request:
+    timeout:
+      jaxrs:
+        clientreqtimeout:
```

Comment:
how to express jaxrs timeout is x, but jaxrs.clientreqtimeout is y, and jaxrs.clientreqtimeout.add is z

below is not a valid yaml:
```
a: 10
  b: 20
```

Reply:
ok. we will modify the key structure.